cmd/format-xl.go

```diff
@@ -776,10 +776,12 @@ func initFormatXL(ctx context.Context, storageDisks []StorageAPI, setCount, driv
 // Make XL backend meta volumes.
 func makeFormatXLMetaVolumes(disk StorageAPI) error {
 	// Attempt to create MinIO internal buckets.
-	err := disk.MakeVolBulk(minioMetaBucket, minioMetaTmpBucket, minioMetaMultipartBucket, minioMetaBackgroundOpsBucket)
-	if err != nil {
-		if !IsErrIgnored(err, initMetaVolIgnoredErrs...) {
-			return err
+	for _, vol := range []string{minioMetaBucket, minioMetaTmpBucket, minioMetaMultipartBucket, minioMetaBackgroundOpsBucket} {
+		err := disk.MakeVol(vol)
```

Comment:
This is incorrect PR @vadmeste you need make MakeVolBulk ignore `errVolumeExists` that's all